integration/templates/generateTemplates.ts

```diff
@@ -0,0 +1,203 @@
+  },
+  {
+    filename: 'cpcFailWelsh',
+    markdown: getRenderedText(TestOutcome.FAIL, personlisationCatCPCFail, Language.WELSH, false)
+  },
+  //Home
+  {
+    filename: 'homePassEnglish',
+    markdown: getRenderedText(TestOutcome.PASS, personlisationCatHomePass, Language.ENGLISH, false)
+  },
+  {
+    filename: 'homePassWelsh',
+    markdown: getRenderedText(TestOutcome.PASS, personlisationCatHomePass, Language.WELSH, false)
+  },
+  {
+    filename: 'homeFailEnglish',
+    markdown: getRenderedText(TestOutcome.FAIL, personlisationCatHomeFail, Language.ENGLISH, false)
+  },
+  {
+    filename: 'homeFailWelsh',
+    markdown: getRenderedText(TestOutcome.FAIL, personlisationCatHomeFail, Language.WELSH, false)
+  },
+  //AM1
+  {
+    filename: 'am1PassEnglish',
+    markdown: getRenderedText(TestOutcome.PASS, personlisationCatAM1Pass, Language.ENGLISH, false)
+  },
+  {
+    filename: 'am1PassWelsh',
+    markdown: getRenderedText(TestOutcome.PASS, personlisationCatAM1Pass, Language.WELSH, false)
+  },
+  {
+    filename: 'am1FailEnglish',
+    markdown: getRenderedText(TestOutcome.FAIL, personlisationCatAM1Fail, Language.ENGLISH, false)
+  },
+  {
+    filename: 'am1FailWelsh',
+    markdown: getRenderedText(TestOutcome.FAIL, personlisationCatAM1Fail, Language.WELSH, false)
+  },
+  //AM2
+  {
+    filename: 'am2PassEnglish',
+    markdown: getRenderedText(TestOutcome.PASS, personlisationCatAM2Pass, Language.ENGLISH, false)
+  },
+  {
+    filename: 'am2PassWelsh',
+    markdown: getRenderedText(TestOutcome.PASS, personlisationCatAM2Pass, Language.WELSH, false)
+  },
+  {
+    filename: 'am2FailEnglish',
+    markdown: getRenderedText(TestOutcome.FAIL, personlisationCatAM2Fail, Language.ENGLISH, false)
+  },
+  {
+    filename: 'am2FailWelsh',
+    markdown: getRenderedText(TestOutcome.FAIL, personlisationCatAM2Fail, Language.WELSH, false)
+  },
+  //ADI3
+  {
+    filename: 'adi3PassEnglish',
+    markdown: getRenderedText(TestOutcome.PASS, personlisationCatADI3Pass, Language.ENGLISH, false)
+  },
+  {
+    filename: 'adi3PassWelsh',
+    markdown: getRenderedText(TestOutcome.PASS, personlisationCatADI3Pass, Language.WELSH, false)
+  },
+  {
+    filename: 'adi3FailEnglish',
+    markdown: getRenderedText(TestOutcome.FAIL, personlisationCatADI3Fail, Language.ENGLISH, false)
+  },
+  {
+    filename: 'adi3FailWelsh',
+    markdown: getRenderedText(TestOutcome.FAIL, personlisationCatADI3Fail, Language.WELSH, false)
+  },
+  //SC
+  {
+    filename: 'scPassEnglish',
+    markdown: getRenderedText(TestOutcome.PASS, personlisationCatSCPass, Language.ENGLISH, false)
+  },
+  {
+    filename: 'scPassWelsh',
+    markdown: getRenderedText(TestOutcome.PASS, personlisationCatSCPass, Language.WELSH, false)
+  },
+  //padi
+  {
+    filename: 'padi',
+    markdown: getRenderedText(TestOutcome.FAIL, personlisationPADI, Language.ENGLISH, true)
+  },
+);
+
+
+templates.forEach((template: any) => {
```

Comment:
`any` -> `{ filename: string; markdown: string | undefined; }`

Reply:
is an extra to the codebase so i was happy with 'any', but will happy change